satellite/satellitedb/overlaycache.go

```diff
@@ -487,7 +465,7 @@ func (cache *overlaycache) UpdateOperator(ctx context.Context, nodeID storj.Node
 		return nil, Error.Wrap(tx.Rollback())
 	}
 
-	updated := getNodeStats(nodeID, updatedDBNode)
+	updated, err := convertDBNode(updatedDBNode)
```

Comment:
should this err be checked before combined and returned? or is it okay to combine nil?

Reply:
it's safe to pass `nil` errors to `errs.Combine`.

Reply:
sure, but we probably shouldn't run tx.Commit if err != nil, right?

Reply:
True. I will fix it.